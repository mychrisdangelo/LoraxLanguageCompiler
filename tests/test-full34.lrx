/* Tests variable assignment */
int main()
{
  int a;
  bool b;
  float c;
  char d;
  a = 33;
  b = true;
  c = 2.483;
  d = 'z';
  print(a);
  print(b);
  print(c);
  print(d);

}
