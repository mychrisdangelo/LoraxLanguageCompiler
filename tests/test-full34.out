33true2.483000z